src/main/java/org/sagebionetworks/web/client/widget/entity/renderer/ChallengeParticipantsWidget.java

```diff
@@ -0,0 +1,93 @@
+package org.sagebionetworks.web.client.widget.entity.renderer;
+
+import java.util.Map;
+
+import org.sagebionetworks.repo.model.UserProfile;
+import org.sagebionetworks.web.client.ChallengeClientAsync;
+import org.sagebionetworks.web.client.SynapseClientAsync;
+import org.sagebionetworks.web.client.utils.Callback;
+import org.sagebionetworks.web.client.widget.WidgetRendererPresenter;
+import org.sagebionetworks.web.client.widget.pagination.DetailedPaginationWidget;
+import org.sagebionetworks.web.client.widget.pagination.PageChangeListener;
+import org.sagebionetworks.web.shared.UserProfilePagedResults;
+import org.sagebionetworks.web.shared.WidgetConstants;
+import org.sagebionetworks.web.shared.WikiPageKey;
+
+import com.google.gwt.user.client.rpc.AsyncCallback;
+import com.google.gwt.user.client.ui.Widget;
+import com.google.inject.Inject;
+
+public class ChallengeParticipantsWidget implements ChallengeParticipantsView.Presenter, WidgetRendererPresenter, PageChangeListener {
```

Comment:
This should be a Team members list widget. It should support a challenge id param (and filtering options) to meet requirements.